Server/src/chat_process/server_process.py

```diff
@@ -0,0 +1,130 @@
+import threading
+import logging
+from socket import socket
+
+from chat_process.message_process import process_chat_commands, broadcast, send_to_client, \
+    send_message_history_server, send_chat_commands, receive_from_client, add_message_history
+from constants import EntryType
+from client_info import ServerRepository, ClientInfo
+from db.db import DataBase
+from db.db_service import DataBaseServices
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def remove_disconnected_user(user_id: int, db_service: DataBaseServices):  # remove client
+    """
+    Remove disconnected user from db and ServerRepository
+    :param user_id: User id from db
+    :param db_service: Service class for db
+    """
+
+    logger.info("Client was disconnect, kicking from server")
+    broadcast(user_id, '{} left!'.format(ServerRepository.clients[user_id].nickname), db_service)
+    db_service.change_offline_user_id(user_id)
+    ServerRepository.clients.pop(user_id)
+
+
+def handle(user_id: int):
+    """
+    Customer message processing
+    :param user_id: User id from db
+    :return: Nothing
+    """
+    db_service = DataBase().get_service()
+    client = ServerRepository.clients[user_id].client
+    while True:
+        message = receive_from_client(client)
+        if message is None:
+            remove_disconnected_user(user_id, db_service)
+            break
+        try:
+            if process_chat_commands(client, message, user_id, db_service):  # processing user message
+                continue
+        except IndexError as e:
+            send_to_client(client, "Error message")
+            continue
+        add_message_history(user_id, 0, message, db_service)
+        message = '{}: {}'.format(ServerRepository.clients[user_id].nickname, message)
+        broadcast(user_id, message, db_service)  # sending messages to users
+
+
+def post_authentication(user_id: int, client: socket, nickname: str, db_service: DataBaseServices):
+    """
+    Allocating a stream to a user and client connection alert
```

Comment:
Post authentication process, adding a user to server repository, and broadcast notification to all users.